content/blog/multitenancy-for-kubernetes-clusters-using-hpe-alletra-6000-and-nimble-storage.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Multitenancy for Kubernetes clusters using HPE Alletra 6000 and Nimble Storage
+date: 2021-06-15T15:00:00.000Z
+author: Michael Mattsson
+authorimage: /img/portrait-2020-192px.png
+tags:
+  - Kubernetes
+  - HPE Alletra
+  - HPE Nimble Storage
+  - HPE Primera
+  - CSI
+---
+In a storage infrastructure economy where IT is under constant pressure to deliver more with less, yet still provide a high standard in data services directly to end users without compromising system or data security.
+
+With the introduction of [HPE CSI Driver for Kubernetes 2.0](https://community.hpe.com/t5/Around-the-Storage-Block/HPE-CSI-Driver-for-Kubernetes-now-available-for-HPE-Alletra/ba-p/7136280) and the software powering HPE Alletra 6000 and Nimble Storage, Hewlett Packard Enterprise introduces multitenancy for Kubernetes clusters accessing persistent volumes on the aforementioned storage arrays.
```

Comment:
In the context of HPE Alletra and Nimble storage, what is the definition of a "multitenancy"
 Does multitenancy refers to the management of multiple tenants where a tenant is a group of users?
 Does multitenancy refers to the management of multiple tenants where a tenant is a user?

To avoid confusion for the readers, I would define here what multitenancy means in the context of HPE Alletra/Nimble storage.